lib/usd/utils/SIMD.h

```diff
@@ -59,7 +60,8 @@
 # define ENABLE_SOME_AVX_ROUTINES 1
 #endif
 
-namespace AL {
+namespace MayaUsd {
```

Comment:
When we make API breaking changes - can we please have a synopsis in the PR description? This stuff is important to know for anyone using these APIs